tests/solidity/test_initialize.py

```diff
@@ -6,36 +6,35 @@
 import elfpy.pricing_models.hyperdrive as hyperdrive_pm
 import elfpy.time as time
 from elfpy.time.time import BlockTime
+from elfpy.math import FixedPoint
 
+# pylint: disable=too-many-instance-attributes
 
-class TestInitialize(unittest.TestCase):
-    """Test case for initializing the market
-
-    .. todo: this is a mirror of the tests in test_markets.py; need to unify
-    """
 
-    # pylint: disable=too-many-instance-attributes
+class TestInitialize(unittest.TestCase):
+    """Test case for initializing the market"""
 
     # TODO: Switching to fixed point or 64 bit float should allow us to increase this to WEI
     # issue #112
-    APPROX_EQ: float = 1e-15
-    contribution: float
-    target_apr: float
-    position_duration: int
+    APPROX_EQ: FixedPoint = FixedPoint(1e-5)
+
+    contribution: FixedPoint
+    target_apr: FixedPoint
+    position_duration: FixedPoint
     alice: elf_agent.Agent
     bob: elf_agent.Agent
     celine: elf_agent.Agent
     hyperdrive: hyperdrive_market.Market
     block_time: BlockTime
     pricing_model: hyperdrive_pm.HyperdrivePricingModel
 
-    def __init__(self, contribution: float = 1_000, target_apr: float = 0.5, position_duration: int = 180, **kwargs):
+    def __init__(self, contribution: float = 1_000.0, target_apr: float = 0.5, position_duration: int = 180, **kwargs):
```

Comment:
more float inputs converted using `FixedPoint(float(x))` right after. maybe this was just the easiest way to convert the test?